patricia/patricia_sparse_test.go

```diff
@@ -417,6 +417,35 @@ func TestParticiaTrie_Delete(t *testing.T) {
 	}
 }
 
+func TestParticiaTrie_DeleteLeakage(t *testing.T) {
```

Comment:
Will do. You might want to add the `Sparse` suffix to some of the other test methods.